src/main/java/org/gridsuite/study/server/networkmodificationtree/dto/BuildStatus.java

```diff
@@ -32,6 +35,10 @@ public static BuildStatus fromApplicationStatus(NetworkModificationResult.Applic
         }
     }
 
+    public static BuildStatus returnHigherSeverityStatus(BuildStatus buildStatus1, BuildStatus buildStatus2) {
+        return Collections.max(List.of(buildStatus1, buildStatus2));
+    }
+
     public boolean isBuilt() {
         return ordinal() >= BUILT.ordinal();
```

Comment:
return this.severityLevel >= 0
Do not use ordinal

Reply:
Done